extensions/applicationinsights-clickanalytics-js/src/common/Utils.ts

```diff
@@ -28,7 +28,7 @@ export type _ExtendedInternalMessageId = number | _eExtendedInternalMessageId |
  */
 export function removeNonObjectsAndInvalidElements(overrideConfig: IClickAnalyticsConfiguration, attributeNamesExpectedObjects: Array<string>): void {
     removeInvalidElements(overrideConfig);
-    for (var i in attributeNamesExpectedObjects) {
+    arrForEach(attributeNamesExpectedObjects, (i) => {
         if (objHasOwn(attributeNamesExpectedObjects, i)) {
```

Comment:
nit: I think we can remove the `objHasOwn` call when using `arrForEach`, but we can do this later